frontends/web/src/routes/account/summary/chart.module.css

```diff
@@ -100,6 +130,14 @@
   position: relative;
 }
 
+@media (max-width: 640px){
+  .chartCanvas {
+    width: 100vw;
+    margin-left: -16px;
+  } 
+}
```

Comment:
Could you try to other way around, have no padding for this view with `<div class="content noSpace">` and add padding/margin for elements that need it?

Reply:
Hm.. I'm not sure if that's the correct solution here. 🤔 

Atm, the chart (along with everything inside the page) is positioned within a container `content padded`. This means, they are inside a container with top, left, and right padding. As we want the chart to be end-to-end (fill up the whole screen horizontally), we don't want this left and right padding, thus we added margin-left of -16px to the chart. I've now changed this margin value to use the css variable: `calc(var(--space-half)*-1)`, negating whatever is the current left and right padding of the page's container (which is just `16px`).

**Now, the chart itself already has no padding and margin, so adding `content noSpace` wont change anything.** We could use this `content noSpace` for the page's container, but this will introduce too changes to other components within the page (adding margin/padding to them).

So I think adding a negative margin **to the chart** is the simplest / imo cleanest solution for it right now. 🙂 pls lmk what you think or if I misunderstood you!

Reply:
👍  for `var(--space-half`

It would go to the edge, but I see, although I think negative margins are nowadays *almost* never needed anymore, it will introduce more changes on the whole view. Negative margin is simplest. 

> Nevertheless, originally, the chart itself already has no padding and margin, so adding content noSpace wont change anything.